src/prefect/server/api/automations.py

```diff
@@ -22,7 +22,7 @@
 from prefect.server.database.dependencies import provide_database_interface
 from prefect.server.database.interface import PrefectDBInterface
 from prefect.server.events import actions
-from prefect.server.events.models import automations
+from prefect.server.events.models import automations as automations_models
```

Comment:
Needed to rename the automations module since otherwise adding it as a param here overrides the endpoint logic below